daml-lf/spec/transaction.rst

```diff
@@ -225,10 +227,11 @@ to construct that instance.
 As of version 10, these fields are included:
 
 * `message Identifier`_ template_id
-* `message VersionedValue`_ value
+* `message VersionedValue`_ arg_versioned
```

Comment:
don’t we need `arg_unversioned` in newer versions?

Reply:
ah nvm, this is `contract_instance` while the other one is not in `ContractInstance`.

Reply:
Since `ContractInstance` is not used any more in Create Node, the only possible usage is alone (for instance in the indexer). In this later case we need the version. 
I add the suffix  `versioned` for the sake of consistency.